os/src/os/ProcessOps.scala

```diff
@@ -218,9 +218,13 @@ case class proc(command: Shellable*) {
 
     builder.directory(Option(cwd).getOrElse(os.pwd).toIO)
 
+    val cmd = 
+      if(scala.util.Properties.isWin) Vector("cmd.exe", "/C") ++ command.flatMap(_.value)
```

Comment:
I don't really understand what's going on here. Why doesn't normal subprocess execution work on windows?

Reply:
I don't understand it either... When running just "dir" command, it just says "Command not found". With this approach I don't get that problem. Everyone recommends to run CMD commands that way..  

I digged this in [Apache Commons Exec](https://github.com/apache/commons-exec/blob/1.3/src/main/java/org/apache/commons/exec/launcher/WinNTCommandLauncher.java#L60). Seems like it is related to "current folder", in code they say:

>         // Use cmd.exe to change to the specified directory before running
>         // the command
>         final CommandLine newCmd = new CommandLine("cmd");
>         newCmd.addArgument("/c");
>         newCmd.addArguments(cmd.toStrings());